### ISSUE TITLE
Coherence on the code snippets in the readme 

### ISSUE THREAD

**Issue by sylvaticus** [JuliaAI/Imbalance.jl#84]
Hello, just to note that some code snippets in the readme assume the previous code snippets (for example the one starting with `using MLJBalancing`), while others are self-contained and don't work if the codes snippets upstair are run (e.g. the TableTransform one, where the function `SMOTE` would give an error  because SMOTE is already defined as the MLJ model name 2 snippets before)

I think you should put them all self-contained or consider them all a continuation of the previous ones.

**Comment by sylvaticus** [comment]
Also, if it's not too complex, it would be nice to have an example where classification without balancing is not good, but then with balancing it improves.


[Part of https://github.com/openjournals/joss-reviews/issues/6310 ]

**Comment by EssamWisam** [comment]
Thank you for making this issue.

> Hello, just to note that some code snippets in the readme assume the previous code snippets (for example the one starting with `using MLJBalancing`), while others are self-contained...

Thank you. I will modify the instance you mentioned so all of them become self-contained.

> Also, if it's not too complex, it would be nice to have an example where classification without balancing is not good, but then with balancing it improves.

Do you mean include example code in the paper itself because I feel it would be a little odd? There are eight examples or so in the [tutorial section](https://juliaai.github.io/Imbalance.jl/dev/examples/) of the docs that demonstrate this.

**Comment by sylvaticus** [comment]
no, I agree with you, not in paper.. my suggestion (and it was just a suggestion) was relative to the readme

(I opened another issue for the JOSS paper, and I will open another one for the documentation, so to keep stuff separated)

**Comment by EssamWisam** [comment]
Have made all code blocks self-contained (except for generating data which I mention at the top) and have also linked to the page demonstrating various examples where resampling improves classification performance. Closing this issue and will reopen in case you have any further feedback.

**Comment by EssamWisam** [comment]
@sylvaticus based on @jbytecode it's preferrable that I ask if it's okay to close the issue first. Could you confirm that it is?

Thank you.

**Comment by sylvaticus** [comment]
Just give me some times on Monday I will double check it... thanks for your understanding..

**Comment by sylvaticus** [comment]
I have created a pull request to separate the code chunk of data generation from those of "standard interface" and I have added "using MLJ" in the MLJBalancing one.

Feel free to merge, modify  or reject the pull request...
